translator/translate/otel/exporter/awsemf/kubernetes.go

```diff
@@ -694,6 +696,33 @@ func getEBSMetricDeclarations(conf *confmap.Conf) []*awsemfexporter.MetricDeclar
 	return metricDeclarations
 }
 
+func getInstanceStoreMetricDeclarations(conf *confmap.Conf) []*awsemfexporter.MetricDeclaration {
+	var metricDeclarations []*awsemfexporter.MetricDeclaration
+	if awscontainerinsight.EnhancedContainerInsightsEnabled(conf) {
+		metricDeclarations = []*awsemfexporter.MetricDeclaration{
+			{
+				Dimensions: [][]string{
+					{"ClusterName"},
+					{"ClusterName", "NodeName", "InstanceId"},
+					{"ClusterName", "NodeName", "InstanceId", "VolumeId"},
+				},
+				MetricNameSelectors: []string{
+					"node_diskio_instance_store_total_read_ops",
+					"node_diskio_instance_store_total_write_ops",
+					"node_diskio_instance_store_total_read_bytes",
+					"node_diskio_instance_store_total_write_bytes",
+					"node_diskio_instance_store_total_read_time",
+					"node_diskio_instance_store_total_write_time",
+					"node_diskio_instance_store_ec2_instance_performance_exceeded_iops",
+					"node_diskio_instance_store_ec2_instance_performance_exceeded_tp",
+					"node_diskio_instance_store_volume_queue_length",
```

Comment:
nit: Is there a reason we aren't using the consts defined in `containerinsightscommon`?